[ROS 2] Error messages if action server exists

If there is an action server on the ROS graph, when rqt_service_caller starts we see errors message about invalid service types.

To reproduce, start turtlesim

    ros2 run turtlesim turtlesim_node

And start rqt_service_caller:

    ros2 run rqt_service_caller rqt_service_caller 

Example error message:

```
[ERROR] [1589841268.191775247] [get_service_class]: Malformed srv message_type: turtlesim/action/RotateAbsolute_GetResult
ServiceCaller.on_refresh_services_button_clicked(): could not get class turtlesim/action/RotateAbsolute_GetResult of service /turtle1/rotate_absolute/_action/get_result. Skipping it.
[ERROR] [1589841268.192278390] [get_service_class]: Malformed srv message_type: turtlesim/action/RotateAbsolute_SendGoal
ServiceCaller.on_refresh_services_button_clicked(): could not get class turtlesim/action/RotateAbsolute_SendGoal of service /turtle1/rotate_absolute/_action/send_goal. Skipping it.
```

I don't necessarily expect the service caller to offer support for calling these "hidden" services, but it should know enough to ignore them without producing errors.

--- a/src/rqt_service_caller/service_caller_widget.py
+++ b/src/rqt_service_caller/service_caller_widget.py
@@ -49,7 +49,7 @@ from python_qt_binding.QtWidgets import QMenu, QTreeWidgetItem, QWidget
 import rclpy
 
 from rqt_py_common.extended_combo_box import ExtendedComboBox
-from rqt_py_common.message_helpers import get_service_class, get_message_class
+from rqt_py_common.message_helpers import get_service_class, get_message_class, SRV_MODE
 from rqt_py_common.topic_helpers import is_primitive_type, get_type_class
 
 class ServiceCallerWidget(QWidget):
@@ -116,12 +116,17 @@ class ServiceCallerWidget(QWidget):
             if len(service_types) > 1:
                 qWarning(
                     'ServiceCallerWidget.on_refresh_services_button_clicked():'
-                    'Topic {} has multiple services available: {}.'.format(
+                    'Service {} has multiple service types available: {}.'.format(
                         service_name, service_types))
                 qWarning(
                     'ServiceCallerWidget.on_refresh_services_button_clicked(): '
                     'using the first option {}'.format(service_types[0]))
 
+            # Skip service files that are not using the "Service mode",
+            # e.g. service types used to implement actions.
+            service_name_tokens = service_types[0].split('/')
+            if len(service_name_tokens) == 3 and service_name_tokens[1] != SRV_MODE:
+                continue
             service_class = get_service_class(service_types[0])
             if service_class is None:
                 qWarning(

